db/schema.rb

```diff
@@ -10,10 +10,9 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2022_06_08_035549) do
+ActiveRecord::Schema.define(version: 2022_06_21_165259) do
```

Comment:
Won't need these schema changes, as well, given that there's no DB change in this PR